src/deployable_example.cpp

```diff
@@ -78,6 +79,8 @@ String queryStateToString (QueryState state) {
       return "LAST_RESPONSE_FAIL";
     case LAST_RESPONSE_UNSURE:
       return "LAST_RESPONSE_UNSURE";
+    case NOT_AUTHENTICATED:
+      return "NOT_AUTHENTICATED";
```

Comment:
this part is not about mirroring in rotation.  please update the PR title to include the functionality.

Reply:
PR title updated